tools/cabana/cabana.cc

```diff
@@ -13,6 +13,7 @@ int main(int argc, char *argv[]) {
   initApp(argc, argv);
   QApplication app(argc, argv);
   app.setApplicationDisplayName("Cabana");
+  app.setWindowIcon(QIcon("./images/cabana.png"));
```

Comment:
You want to use qrc to load the file, otherwise it won't work if your current working directory is wrong.